### DISCUSSION TITLE
OpenSSL 3.4.0

### DISCUSSION THREAD

**Discussion by mattcaswell** [openssl/openssl#25767] [Announcements]
OpenSSL 3.4.0 has been released. You can find more details about this release in the [release notes](https://github.com/openssl/openssl/blob/openssl-3.4.0/NEWS.md).

<hr /><em>This discussion was created from the release <a href='https://github.com/openssl/openssl/releases/tag/openssl-3.4.0'>OpenSSL 3.4.0</a>.</em>

**Comment by 13ilya-old** [comment]
Where is the previously promised server-side QUIC and when to expect it now?

**Reply by t-j-h** [reply]
https://github.com/openssl/openssl/tree/feature/quic-server is the current status of the QUIC server work for any early adopters that want to work with the pre-release code. It is being actively worked on.

It is now planned to be released as part of OpenSSL-3.5 which will be out in April 2025.

**Comment by Musabdra** [comment]
Please somebody help with the link that will give me access to download the tar file  of openssl in my wsl. 
The link in the site openssl.org/releases is not working in my pc. 
Or I should say I need guidance on how to install and build openssl in wsl.
Somebody help me please!
